qualtran/_infra/gate_with_registers.py

```diff
@@ -172,17 +175,21 @@ def add_my_tensors(
         incoming: Dict[str, 'SoquetT'],
         outgoing: Dict[str, 'SoquetT'],
     ):
-        from qualtran.cirq_interop._cirq_to_bloq import _add_my_tensors_from_gate
-
-        _add_my_tensors_from_gate(
-            self,
-            self.signature,
-            self.short_name(),
-            tn=tn,
-            tag=tag,
-            incoming=incoming,
-            outgoing=outgoing,
-        )
+        try:
+            # First try to add tensors via decomposition.
+            Bloq.add_my_tensors(self, tn, tag, incoming=incoming, outgoing=outgoing)
```

Comment:
super().add_my_tensors(...)? or maybe we should factor out the body of the default `Bloq.add_my_tensors` into a descriptively named, private free function

Reply:
removed the `add_my_tensors` override. The only benefit now was if `GateWithRegisters` doesn't define a decomposition but defines a `_unitary_` then the fallback would take care of it, but it's not a common use case so I've removed the override.